lms/djangoapps/branding/api.py

```diff
@@ -220,6 +242,58 @@ def _footer_legal_links():
     ]
 
 
+def _footer_business_links():
+    """Return the edx links to display in the footer. """
+    platform_name = configuration_helpers.get_value('platform_name', settings.PLATFORM_NAME)
+
+    return [
+        {
+            "name": link_name,
+            "title": link_title,
+            "url": link_url,
+        }
+        for link_name, link_url, link_title in [
+            ("about", marketing_link("ABOUT"), _("About")),
+            ("enterprise", marketing_link("ENTERPRISE"),
+             _("{platform_name} for Business").format(platform_name=platform_name)),
```

Comment:
Nit: this line seems difficult for the community to control, since they likely won't want to call it "FOO for Business". I can't think of a cleaner way to do it right now, so I'm okay with merging as is and someone can fix it in the future when it becomes clear what to do.